model/inline/src/main/scala/aqua/model/inline/state/Exports.scala

```diff
@@ -59,18 +124,91 @@ trait Exports[S] extends Scoped[S] {
 object Exports {
   def apply[S](implicit exports: Exports[S]): Exports[S] = exports
 
+  // Get last linked VarModel
+  def getLastHelper[T](name: String, state: Map[String, ValueModel], ex: VarModel => T): Option[T] = {
```

Comment:
Why pass `ex` here and then pass identity at call site?
Let a call site do `map` on option if it is needed